src/editor.c

```diff
@@ -1886,6 +1867,53 @@ static gchar *find_calltip(const gchar *word, GeanyFiletype *ft)
 			g_ptr_array_free(constructor_tags, TRUE);
 		}
 	}
+	return tags;
+}
+
+
+static void update_tag_for_construtor_calltip(const gchar *word, TMTag *tag, const gchar **tag_name,
```

Comment:
I would call this `update_tag_name_and_scope_for_calltip()` or something similar because `update_tag` feels like it changes the `TMTag`.

Reply:
Done, thanks.